<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>certificate_detail_widget</class>
 <widget class="QWidget" name="certificate_detail_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Serial Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>-10</y>
     <width>1141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font-size: 32px;
    font-weight: bold;
    color: white;
background-color: #16A085;
    padding: 12px 20px;
    border-bottom: 2px solid #1A252F;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Thông tin chứng chỉ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>-248</x>
     <y>108</y>
     <width>1291</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image/resource.qrc">:/new/image/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Issuer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>295</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Ngày bắt đầu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Public Key Info:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Signature Algorithm:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Signature Value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>335</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Public Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>295</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Hết hạn:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image/resource.qrc">:/new/image/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_version">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>134</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_serial">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>173</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issuer">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>213</y>
     <width>891</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_endDay">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>299</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_startday">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>298</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pubkeyInfo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>493</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_signatureAlgorithm">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>533</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
      font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>252</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
    color: #2C3E50;
    font-size: 25px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_subject">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>255</y>
     <width>871</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: #0b2b42;
    font-size: 18px;
    padding: 4px 8px;
</string>
   </property>
   <property name="text">
    <string>###</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>971</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>969</width>
      <height>99</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_pubkey">
       <property name="styleSheet">
        <string notr="true">    color: #2C3E50;
    font-size: 18px;
    padding: 4px 8px;
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>971</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>969</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_signatureValue">
       <property name="styleSheet">
        <string notr="true">
    color: #2C3E50;
    font-size: 18px;
    padding: 4px 8px;
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>label_18</zorder>
  <zorder>label_16</zorder>
  <zorder>label_8</zorder>
  <zorder>label_15</zorder>
  <zorder>label_17</zorder>
  <zorder>label_20</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>label_25</zorder>
  <zorder>label_28</zorder>
  <zorder>label_32</zorder>
  <zorder>label</zorder>
  <zorder>lb_version</zorder>
  <zorder>lb_serial</zorder>
  <zorder>lb_issuer</zorder>
  <zorder>lb_endDay</zorder>
  <zorder>lb_startday</zorder>
  <zorder>lb_pubkeyInfo</zorder>
  <zorder>lb_signatureAlgorithm</zorder>
  <zorder>label_19</zorder>
  <zorder>lb_subject</zorder>
  <zorder>scrollArea</zorder>
  <zorder>scrollArea_2</zorder>
 </widget>
 <resources>
  <include location="image/resource.qrc"/>
 </resources>
 <connections/>
</ui>
